src/Training.TruckWorld.Clone.Backend/TruckWorld.Infrastructure/Common/Notifications/Services/EmailTemplateService.cs

```diff
@@ -0,0 +1,64 @@
+﻿using FluentValidation;
+using Microsoft.EntityFrameworkCore;
+using System.Linq.Expressions;
+using TruckWorld.Application.Common.Models.Querying;
+using TruckWorld.Application.Common.Notifications.Services;
+using TruckWorld.Application.Common.Querying.Extensions;
+using TruckWorld.Domain.Entities;
+using TruckWorld.Domain.Enums;
+using TruckWorld.Persistence.Repositories.Interfaces;
+
+namespace TruckWorld.Infrastructure.Common.Notifications.Services;
+/// <summary>
+/// created crud methods for emailTemplate
+/// </summary>
+public class EmailTemplateService : IEmailTemplateService
```

Comment:
should be one row in each class or method . In here , should be row after namespace